packages/angular/src/rule-builder/rule.component.ts

```diff
@@ -0,0 +1,127 @@
+import { Component, EventEmitter, HostBinding, Input, Output } from '@angular/core';
+import { ListItem } from 'carbon-components-angular';
+
+@Component({
+  selector: 'ai-rule',
+  template: `
+    <ng-container *ngIf="isRuleGroup">
+      <ai-rule-builder-group-logic [id]="rule.id" [(selected)]="rule.groupLogic">
+      </ai-rule-builder-group-logic>
+      <ng-container *ngFor="let r of rule.rules; let i = index">
+        <ai-rule
+          (addRule)="addRule.emit($event)"
+          (removeRule)="removeRule.emit($event)"
+          [columns]="columns"
+          [columnOperands]="columnOperands"
+          [(rule)]="rule.rules[i]"
```

Comment:
Can we just use `r`?

Reply:
No, not for double binding. Angular.
Story works fine, build fails.